pkg/solana/chainreader/client_wrapper.go

```diff
@@ -25,15 +25,14 @@ func (w *RPCClientWrapper) GetMultipleAccountData(ctx context.Context, keys ...s
 		return nil, err
 	}
 
-	bts := make([][]byte, len(result.Value))
-
-	for idx, res := range result.Value {
+	var accounts []*rpc.Account
+	for _, res := range result.Value {
 		if res == nil || res.Data == nil || res.Data.GetBinary() == nil {
```

Comment:
Should we also len check the binary here?
```suggestion
		if res == nil || res.Data == nil || res.Data.GetBinary() == nil || len(res.Data.GetBinary()) == 0 {
```

Reply:
I would leave that up to the caller to decide, don't want to handle it at rpc call level